app/Http/Controllers/Adm/Mship/Feedback.php

```diff
@@ -12,6 +13,44 @@
 
 class Feedback extends \App\Http\Controllers\Adm\AdmController
 {
+    public function getNewForm()
+    {
+        $question_types = Type::all();
+        $new_question = new Question();
+
+        return $this->viewMake('adm.mship.feedback.new')
+            ->with('question_types', $question_types)
+            ->with('new_question', $new_question);
+    }
+
+    public function postNewForm(UpdateFeedbackFormRequest $request)
+    {
+        $new_ident = isset($_POST['ident']) ? $_POST['ident'] : null;
+        $new_name = isset($_POST['name']) ? $_POST['name'] : null;
+        $new_contact = isset($_POST['contact']) ? $_POST['contact'] : null;
+        if ($new_ident == null) {
+            return Redirect::back()
+                ->withInput($request->input())
```

Comment:
Could some of this validation code be moved into the UpdateFeedbackFormRequest, or a new Request class made for it? You can process the setting of new name/contact in the `getValidatorInstance()` of the request